### ISSUE TITLE
When trying to create a flat map: ValueError: invalid literal for int() with base 10: ''

### ISSUE THREAD

**Issue by svioletg** [xtotdam/dynmap-assemble#6]
Whenever I try to create an image for the flat map, I get this:

```
Choose world (enter number)
   1:  world
   2:  world_nether
   3:  world_the_end
> 1
Choose map (enter number)
   1:  ct
   2:  flat
   3:  t
> 2
Choose background color (enter number)
   1:  transparent ()
   2:  overworld (light blue)
   3:  nether (dark red)
   4:  the_end (dark violet)
   5:  black ()
   6:  white ()
> 6
Assembling map 'flat' of world 'world' located at '/Users/macmini/mc/bcr-mc/plugins/dynmap/web/tiles/world/flat' using 'white ()' as background color
Collecting tile locations...
Traceback (most recent call last):
  File "/Users/macmini/mc/bcr-mc/plugins/dynmap/web/tiles/dynmap-assemble.py", line 99, in <module>
    h, v = map(int, hv.split('_'))
ValueError: invalid literal for int() with base 10: ''
```

If I choose `t` it seems to work out fine, but whenever I try `flat` it just gives me that error. I installed pillow and tqdm, not really sure how to resolve this?

**Comment by xtotdam** [comment]
Hi!
While choosing map my script merely looks at which folders are present in a directory where DynMap stores tiles.
Maybe it didnt find them at all?
Try `/dynmap fullrender world:flat` in your server's console to rebuild full cache of tiles for `flat` map. If I don't recall command syntax correctly, consult https://github.com/webbukkit/dynmap/wiki/Commands

**Comment by svioletg** [comment]
I added a bunch of print commands to the for loop where things are going wrong, and it looks like it got hung up on my Mac's .DS_Store folder. I checked and the same folder is in the `t` folder, which seemingly had no problems generating a map, so I don't know why it only got caught on it in the flat folder. Regardless, I added:
```
if t=='.DS_Store':
    break
```
right before `hv = t.split('.')[0]` where it was having trouble and things seem to be fixed!
